.github/workflows/nightly-pr-checks.yaml

```
@@ -0,0 +1,212 @@
+name: PR checks
+
+on: # yamllint disable-line rule:truthy
+  schedule:
+    - cron: '0 0 * * *' # run at midnight daily
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  GO111MODULE: "on"
+  MINIKUBE_WANTUPDATENOTIFICATION: false
+  MINIKUBE_WANTREPORTERRORPROMPT: false
+  SDK_VERSION: "1.25.0"
+  K8S_VERSION: "1.25.3"
+  GO_VERSION: "^1.20"
+  MINIKUBE_VERSION: "1.28.0"
+  TEST_ACCEPTANCE_CLI: "kubectl"
+  TEST_RESULTS: "out/acceptance-tests"
```

some of this may be removed

```suggestion
env:
  GO111MODULE: "on"
  GO_VERSION: "^1.20"
  TEST_ACCEPTANCE_CLI: "kubectl"
```